.vscode/tasks.json

```diff
@@ -18,6 +18,13 @@
             "problemMatcher": [
                 "$tsc"
             ]
+        },
+        {

```

Comment:
Allows debugging single tests from VS Code by pressing F5 in the testcase's editor window